content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
 
-At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
-specific report. Filters show information that matches the specific filter parameters. For example,
-when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 365 days
-appear. All reports have the option to add a custom filter, and each have different default custom
-filter options. Select the parameters, then click the :guilabel:`Apply` button.
+Filters
+=======
 
-.. image:: reporting/custom-filter.png
-   :align: center
-   :alt: Report dashboard view.
+At the top of each report, the default filters are shown inside the :guilabel:`Search` box.
 
-Clicking on :guilabel:`Favorites` displays three options: :guilabel:`Save the current search`,
-:guilabel:`Add the search to the dashboard`, or :guilabel:`Insert the search to a Google
-spreadsheet`.
+Click the :guilabel:`drop-down arrow` in the search box to display the available
+:guilabel:`filters`. Filters show information that matches the specific filter parameters.
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+.. example::
+   The :guilabel:`Work Entries Analysis` report has two default filters, the :guilabel:`Current
+   month:(Month)(Year)` filter, and the :guilabel:`Validated` filter. The :guilabel:`Payroll` report
+   only has one default filter, the :guilabel:`Last 365 Days Payslip` filter.
+
+   .. image:: reporting/custom-filter.png
+      :align: center
+      :alt: Filters enabled for the Work Entries Analysis report.
+
+All reports can include custom filters or group information by different metrics (employee,
+department, company, etc.). Some reports have the option to compare the current report to the
+previous time period or year (a :guilabel:`Comparison` option).
+
+Click on a parameter to select it and activate it. The report is immediately updated with the new
+parameters.
+
+The updated report can be set as a favorite report, meaning the parameters are stored for quick
+access in the future. First, click :guilabel:`Save the current search` under the
+:guilabel:`Favorites` section, and two options and a :guilabel:`Save` button appear.
+
+To set the current report as the default configuration when the report is accessed, check the box
+next to :guilabel:`Default filter`. If the current report should be accessible to everyone in the
+database, check the box next to :guilabel:`Share`.
+
+Finally, click :guilabel:`Save`, which saves the currently configured report, and it now appears
+beneath :guilabel:`Favorites`.
+
+Default reports
+===============
 
 Payroll
 -------
 
-Click on :menuselection:`Payroll --> Reporting --> Payroll` to display the :guilabel:`Payroll
-Analysis` report.
+Click on :menuselection:`Payroll app --> Reporting --> Payroll` to display the :guilabel:`Payroll
+Analysis` report. This report shows all the payslips generated in the last 365 days.
```

Comment:
```suggestion
Analysis` report. This report shows all the payslips generated in the last 365 days.
```
this would be a good place to mention the default filter again - just in case readers skip to this portion